### PR TITLE
Regional Partners now have workshop info in the table, and have cours…

### PR DESCRIPTION
**Pull request by mehalshah**
…e as a column as well

Quick migration - columns aren't getting backfilled. Table has 12,921 rows

### CODE REVIEW

File: dashboard/db/migrate/20170117185757_add_course_workshop_info_to_regional_partner_district_mapping.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddCourseWorkshopInfoToRegionalPartnerDistrictMapping < ActiveRecord::Migration[5.0]
+  def change
+    add_column :regional_partners_school_districts, :course, :string
+    add_column :regional_partners_school_districts, :workshop_info, :string
```
Comment (aoby) [inline dashboard/db/migrate/20170117185757_add_course_workshop_info_to_regional_partner_district_mapping.rb]: What is `workshop_info`? Will that be just the date string describing the workshop? Please add a `comment:` here.

Should we have a workshop id field?

Comment (mehalshah) [reply dashboard/db/migrate/20170117185757_add_course_workshop_info_to_regional_partner_district_mapping.rb]: Date string describing the workshop.

Comment (ashercodeorg) [reply dashboard/db/migrate/20170117185757_add_course_workshop_info_to_regional_partner_district_mapping.rb]: Why not name it `workshop_date` or something similar?

File: dashboard/app/models/regional_partners_school_district.rb [hunk 2]
```diff
@@ -4,6 +4,8 @@
 #
 #  regional_partner_id :integer          not null, primary key
 #  school_district_id  :integer          not null, primary key
+#  course              :string(255)
```
Comment (ashercodeorg) [inline dashboard/app/models/regional_partners_school_district.rb:7]: What are the possible choices for choice? Should this model include some validation on this field? Etc.

Comment (mehalshah) [reply dashboard/app/models/regional_partners_school_district.rb:7]: I'll add validation and rename the string

**Pull request merged**